iron-overlay-manager.html

```diff
@@ -78,10 +78,58 @@
     },
 
     /**
+     * Brings the overlay at the specified index to the front.
+     * @param {number|Element} i
+     */
+    bringToFront: function(i) {
+      // If not a number, assume i is the overlay.
+      if (typeof i !== 'number') {
+        i = this._overlays.indexOf(i);
+      }
+      var overlay = this._overlays[i];
+      var lastI = this._overlays.length - 1;
+      // If already the top element, return.
+      if (!overlay || i === lastI) {
+        return;
+      }
+      // Update z-index to be on top.
+      var minimumZ = Math.max(this.currentOverlayZ(), this._minimumZ);
+      if (this._getZ(overlay) <= minimumZ) {
+        this._applyOverlayZ(overlay, minimumZ);
+      }
+
+      // Shift other overlays behind the new on top.
+      while (i < lastI) {
+        this._overlays[i] = this._overlays[i + 1];
+        i++;
+      }
+      this._overlays[lastI] = overlay;
+    },
+
+    /**
+     * Adds the overlay and updates its z-index if it's opened, or removes it if it's closed.
+     * Also updates the backdrop z-index.
+     * @param {Element} overlay
+     */
+    trackOverlay: function(overlay) {
```

Comment:
This seems like a good method to have, but `track` is an ambiguous verb to me. Given that the behavior is conditional, something like `addOrRemoveOverlay` might be appropriate. WDYT?